scripts/compare_dj30_nas100_configs.py

```
@@ -0,0 +1,158 @@
+            print(f"  {s:<15} {n/total_b:>6.1%}")
+    print("Scale factors (1R, scale, n):")
+    for s, info in r["scale_info"].items():
+        tag = " [fallback]" if info["fell_back"] else ""
+        print(f"  {s:<15} 1R=${info['implied_1r']:>7,.2f}  scale={info['scale']:>6.3f}  n={info['n_trades']}{tag}")
+    return r
+
+
+def main() -> int:
+    print("Portfolio MC comparison — DJ30 + NAS100 config sweep")
+    print(f"dd_protection: trigger={DD_TRIGGER:.3f} ({DD_TRIGGER:.1%}) / scale={DD_SCALE}")
+    print(f"Guardian + Aegis panels: held constant")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)